openstack/download.sh

```diff
@@ -4,12 +4,13 @@ set -euo pipefail
 SCRIPT_DIR="$(cd "$(dirname "$0")" && pwd)"
 source "$SCRIPT_DIR/common.sh"
 
-if [[ "$STATION_STATUS" != "RUNNING" ]]; then
+if [[ "$STATION_STATUS" != "ACTIVE" ]]; then
   echo "Station is not running. Run 'station start' to start it"
   exit 1
 fi
 
 scp \
   -A \
   -o "UserKnownHostsFile=/dev/null" \
-  "${EIRINI_STATION_USERNAME}@${STATION_IP}:$1" "$2"
+  "ccloud@${STATION_IP}:$1" "$2"
```

Comment:
Use the `VMUSER` env var instead of hardcoding, so that we are consistent

Reply:
somehow this was already corrected for me... hopefully the new commit will show that